src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((item) => item.trim() !== '')
+    .reduce((stylesObject, item) => {
+      const pair = item.split(':');
+
+      const key = pair[0].trim();
+      const value = pair[1].trim();
+
+      stylesObject[key] = value;
+
+      return stylesObject;
+    }, {});
```

Comment:
The function `convertToObject` is correctly using string and array methods to transform the CSS string into an object. However, it's important to note that the code does not handle cases with multiple colons within the property values or multiple semicolons without any styles between them, which could be considered for a more robust implementation. Additionally, the chained methods `.split`, `.filter`, and `.reduce` should be indented on the same level to improve readability.